tests/e2e/vLLM/test_vllm.py

```diff
@@ -158,6 +168,9 @@ def test_vllm(self):
         self.tear_down()
 
     def tear_down(self):
+        import torch
+        torch.cuda.empty_cache()
```

Comment:
Can we remove this? `torch.cuda.empty_cache()` doesn't do anything useful except free gpu memory for other programs. I see no reason why this would be necessary for tear down

Reply:
ok